source/stylesheets/refills/unstyled/_search-bar.scss

```diff
@@ -0,0 +1,25 @@
+.search-bar {
+  width: 100%;
+
+  form {  
+    position: relative;
+
+    input[type=search] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3